src/ol_infrastructure/infrastructure/aws/dns/__main__.py

```diff
@@ -14,6 +14,16 @@
     opts=mitxpro_legacy_opts,
 )
 
+ocw_legacy_dns_name = "ocw-legacy.ocw.mit.edu"
```

Comment:
```suggestion
ocw_legacy_dns_name = "old.ocw.mit.edu"
```
This is already set up as old.ocw.mit.edu

Reply:
Gonna leave this for now so I can resolve when we go live.